gulpfile.babel.js

```diff
@@ -154,6 +154,18 @@ gulp.task('html', () => {
       removeStyleLinkTypeAttributes: true,
       removeOptionalTags: true
     })))
+    .pipe($.if('*.html', $.minifyInline({
+            js: {
+                output: {
+                    comments: true
+                }
+            },
+            jsSelector: 'script[type!="text/x-handlebars-template"]',
+            css: {
+                keepSpecialComments: 1
+            },
+            cssSelector: 'style[do-not-minify!="true"]'
```

Comment:
I agree. Just for context, both this and the handlebar exclusion are straight up from [the `gulp-minify-inline` README](https://www.npmjs.com/package/gulp-minify-inline).